scripts/build-rosinstall.py

```diff
@@ -40,6 +40,33 @@ def gh_issue_to_datetime(url_issue):
     return created_at
 
 
+def _time_machine(packages,
+                  dt,
+                  distro,
+                  deps_only=True):
+    # type: (List[str], str, str) -> Dict[str, Dict[str, str]]
+    cmd = [BIN_TIME_MACHINE, dt, distro] + packages
+    cmd += ['--deps', '--tar']
+    if deps_only:
+        cmd += ['--deps-only']
+    logger.debug("executing command: %s", ' '.join(cmd))
+
+    try:
+        res = subprocess.run(cmd,
+                             check=True,
+                             stdout=subprocess.PIPE)
+        contents = res.stdout.decode('utf-8')
+    except subprocess.CalledProcessError as err:
+        logger.warning("time machine failed (return code: %d) for bug [%s]",
+                       err.returncode, fn_bug_desc)
+        return
+
+    # updated repository names
+    contents = contents.replace('geometry_experimental', 'geometry2')
```

Comment:
Can we do this unconditionally?

Reply:
Is there any case when we wouldn't want to do this? I'm not sure.

Reply:
I don't think there is actually, thinking some more about it.

`geometry_experimental` (as in: the repository) doesn't exist any more. And it won't come back either.

So all references to that repository should be updated to point to its current name, which would be `geometry2`.

Could we perhaps print a notice that `build-rosinstall.py` has renamed entries in the rosinstall file?

(perhaps too difficult/involved) could it add a comment to the `.rosinstall` file?

Reply:
Sure. We could print a notice and add a comment to the top of the `.rosinstall` file.

Reply:
Done. Do you mind checking the wording of the message, @gavanderhoorn?